src/test/java/cow/farm/repo/CowHerdServiceTest.java

```diff
@@ -0,0 +1,25 @@
+package cow.farm.repo;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+
+public class CowHerdServiceTest {
+
+	@Test
+	public void shouldAddCowToListAndReturnHashMap() {
+		// given
+		CowHerdService cow = new CowHerdService();
+		cow.makeCow(Cow.Breed.ANGUS);
+		cow.makeCow(Cow.Breed.ANGUS);
+		cow.makeCow(Cow.Breed.RANDALL);
+		cow.makeCow(Cow.Breed.ABONDANCE);
+		cow.makeCow(Cow.Breed.REDPOOL);
+		// when
+		HashMap breedQuantity = cow.getHerdMap();
+		// then
+		Assert.assertEquals(breedQuantity.get(Cow.Breed.ANGUS), 2);
```

Comment:
dobry test, tylko pamiętaj o dokładności, jak dodajesz 2x R... to też przetestuj :)